src/GoogleAdWordsEventForwarder.js

```diff
@@ -393,6 +533,29 @@
                     return 'Can\'t initialize forwarder: ' + name + ', conversionId is not defined';
                 }
 
+                // https://go.mparticle.com/work/SQDSDKS-6165
+                if (window.gtag && forwarderSettings.enableGtag === 'True') {
+                    if (forwarderSettings.consentMappingSDK) {
```

Comment:
`consentMappingWeb` given [this commit](https://git.corp.mparticle.com/mParticle/mPServer/pull/26822/commits/1b865d249bea8bb40097b356c09ec3a9d81da050)